src/main/java/org/candlepin/model/CloudProfileFacts.java

```diff
@@ -32,7 +32,7 @@ public enum CloudProfileFacts {
     DMI_SYSTEM_UUID(Consumer.Facts.DMI_SYSTEM_UUID),
     MEMORY_MEMTOTAL(Consumer.Facts.MEMORY_MEMTOTAL),
     OCM_UNITS(Consumer.Facts.OCM_UNITS),
-    UNAME_MACHINE(Consumer.Facts.UNAME_MACHINE),
+    ARCHITECTURE(Consumer.Facts.ARCHITECTURE),
```

Comment:
The renaming of CloudProfileFacts.UNAME_MACHINE to CloudProfileFacts.ARCHITECTURE wasn't outlined in the design, but it seems like a good change to make so that it aligns with the Consumer.facts name. Also, CloudProfileFacts.UNAME_MACHINE was not being used anywhere currently.

Reply:
Seems like a great idea to me